ts/features/bonus/cgn/screens/onboarding/CgnInformationScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import I18n from "../../../../../i18n";
+import { Dispatch } from "../../../../../store/actions/types";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { availableBonusTypesSelectorFromId } from "../../../bonusVacanze/store/reducers/availableBonusesTypes";
+import { ID_CGN_TYPE } from "../../../bonusVacanze/utils/bonus";
+import BonusInformationComponent from "../../../common/components/BonusInformationComponent";
+import { navigateBack } from "../../../../../store/actions/navigation";
+
+export type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+/**
+ * This Screen shows all the information about the cgn program, with the rules and t&c.
+ */
+const CgnInformationScreen: React.FunctionComponent<Props> = (props: Props) => {
+  const onConfirm = () => props.userActivateCgn();
+
+  return (
+    <>
+      {props.bonus ? (
+        <BonusInformationComponent
+          primaryCtaText={I18n.t("bonus.cgn.cta.activeBonus")}
+          bonus={props.bonus}
+          onConfirm={onConfirm}
```

Comment:
I don't know if onConfirm will be used few times, but at the moment you can void to create a new function
```suggestion
          onConfirm={props.userActivateCgn}
```

Reply:
Still don't know at this time if we may include a check if the CGN is yet active and we'll need to show an alert to the user such we actually do on other bonuses